en/concept/dronecode_architecture.md

```diff
@@ -1,17 +1,53 @@
-# PX4 Platform Hardware/Software Architecture
+# PX4 System Architecture
 
-The diagram below provides a forward-looking high level overview of a full PX4 onboard and offboard stack.
-The left hand side shows one possible hardware configuration with a *flight controller* (light blue) connected to a *perception computer* (dark blue) via [RTPS](../middleware/micrortps.md).
-The perception computer provides vision control and object avoidance using a camera sensor array, and has a separate payload camera.
-
-The right hand side of the diagram shows the end-to-end software stack.
-The stack "approximately" aligns horizontally with the hardware parts of the diagram, and is colour-coded to show which software is running on the flight controller and which on the companion computer.
+The sections below provide high-level overview of the PX4 hardware and software stack for two "typical" PX4 systems; one that has just a flight controller, and another that has a flight controller and a mission computer (also known as a "companion computer"). 
 
 :::note
 The [PX4 Architectural Overview](../concept/architecture.md) provides information about the flight stack and middleware.
 Offboard APIs are covered in [ROS](../ros/README.md) and [MAVSDK](https://mavsdk.mavlink.io/develop/en/index.html).
 :::
 
-![PX4 Platform architecture](../../assets/diagrams/dronecode_platform_architecture.jpg)
 
-<!-- The drawing is on draw.io: https://drive.google.com/file/d/14sgSpcs7NcBatW-qn0dLtyMHvwNMSSlm/view?usp=sharing. Request access from dev team. -->
+## Flight Controller (only)
+
+The diagram below provides a high level overview of a typical "simple" PX4 system based around a flight controller.
+
+![PX4 architecture - FC only system](../../assets/diagrams/px4_arch_fc.svg)
+
+<!-- Source for drawing: https://docs.google.com/drawings/d/1_2n43WrbkWTs1kz0w0avVEeebJbfTj5SSqvCmvSOBdU/edit -->
+
+The hardware consists of 
+- [Flight controller](../flight_controller/README.md) (running the PX4 flight stack). This often includes internal IMUs, compass and barometer.
+- [Motor ESCs](../peripherals/esc_motors.md) connected to [PWM outputs](../peripherals/pwm_escs_and_servo.md) or a [UAVCAN](../peripherals/uavcan_escs.md) bus (UAVCAN allows two-way communication, not single direction as shwown).
```

Comment:
I'd keep this a bit broader as there's no only the 2 protocols, or motors: Actuators (motors and servos).
Also on the diagram

Reply:
OK, so add Dshot and also say "or other protocols"?

Reply:
Yes, but there's also more buses, like I2C or serial.

Reply:
Ah, I'd only seen ESC connected to PWM or UAVCAN - had assumed I2C or serial were never used for this

Reply:
Fixed.